packages/perseus/src/widgets/radio/__stories__/option-status.stories.jsx

```diff
@@ -22,6 +22,8 @@ export const CheckedCorrectPreviouslyAnswered = (
             checked={true}
             correct={true}
             previouslyAnswered={true}
+            reviewMode={true}
```

Comment:
OptionStatus is only used by Choice and only once, so I took some of the logic for OptionStatus that was in Choice and moved it to OptionStatus.